src/neynar-api/v2/client.ts

```diff
@@ -927,6 +928,48 @@ export class NeynarV2APIClient {
     return response.data;
   }
 
+    /**
+   * Retrieves information about multiple casts using an array of their hashes. This method is useful
+   * for fetching details of several casts at once, identified by their unique hashes. Optional parameters
+   * allow for adding viewer context to the cast objects to show whether the viewer has liked or recasted
+   * the cast and sorting the casts based on different criteria.
+   *
+   * @param {Array<string>} castsHashes - An array of strings representing the hashes of the casts
+   *   to be retrieved.
+   * @param {Object} [options] - Optional parameters for the request.
+   * @param {number} [options.viewerFid] - Adds viewer context to cast objects to indicate whether the viewer has liked or recasted the cast.
+   * @param {'trending' | 'likes' | 'recasts' | 'replies' | 'recent'} [options.sortType] - Optional parameter to sort the casts based on different criteria such as trending, likes, recasts, replies, or recent.
+   *
+   * @returns {Promise<CastsResponse>} A promise that resolves to a `CastsResponse` object,
+   *   containing information about the requested casts.
+   *
+   * @example
+   * // Example: Fetch information about a casts conversation using a given hash and type
+   *
+   *
+   * client.lookupCastConversation(https://warpcast.com/rish/0x9288c1,
+   *  type: 'url',
```

Comment:
CastParamType.Url should be used here. We are doing this so new users can get an idea of how to use our SDK

Search for `CastParamType` in this file, you'll get an example on how to write the examples for the enums